packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedFieldMessageEmbed.ts

```diff
@@ -0,0 +1,155 @@
+import { MessageEmbed, MessageEmbedOptions } from 'discord.js';
+
+import { PaginatedMessage } from './PaginatedMessage';
+
+/**
+ * This is a utility of {@link PaginatedMessage}, but paginating only the field of an embed.
+ * You must either use this class directly or extend it.
+ *
+ * @example
+ * ```typescript
+ * const { PaginationFieldMessageEmbed } = require('@sapphire/discord.js-utilities');
+ *
+ * new PaginationFieldManager()
+ *    .setTitleField('')
+ *    .setTemplate({ ...options })
```

Comment:
Where is options coming from?

Reply:
The MessageEmbed object options.
In the method to configure that I explain with examples.

Reply:
No what I mean is, define those options in the example.